### PR TITLE
first croatian story

### PR DESCRIPTION
**Pull request by lord015r15**

### CODE REVIEW

**Comment by lord015r15** [comment]
Please review this, @casey-collab

File: croatian/sljez.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Jednog jutra, probudis se u cudnoj sobi za sidovima od sljezovih slatkisa.
```
Comment (casey-collab) [inline croatian/sljez.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: croatian/sljez.md (hunk 1 above)
Comment (lord015r15) [inline croatian/sljez.md]: I have added link to my page. Everything should be fine now.

**Comment by lord015r15** [comment]
Please review this again @casey-collab

File: language.md [hunk 2]
```diff
@@ -53,3 +53,4 @@ Maligayang Bati! [Philippines](Philippines/salamat.md)
 Εάν μιλάτε ελληνικά, συνεχίστε [εδώ](greek/marshmallow.md)
 
 Ja runā latviski, [dodies šeit](./latvian/zefirs.md).
+Ako govoris hrvatski, [klikni ovdje](croatian/sljez.md)
```
Comment (casey-collab) [inline language.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by lord015r15** [comment]
Please try again  @casey-collab

**Comment by casey-collab** [comment]
Looks great! :hibiscus:

**Pull request merged**